lib/bugsnex/plug.ex

```diff
@@ -75,4 +77,21 @@ defmodule Bugsnex.Plug do
     {:ok, hostname} = :inet.gethostname
     to_string(hostname)
   end
+
+  defp filter_parameters(params) do
+    do_filter(params, Application.get_env(:bugsnex, :filter_params, @default_filter_params))
+  end
+
+  defp do_filter(%{__struct__: mod} = struct, _params) when is_atom(mod), do: struct
+  defp do_filter(%{} = map, params) do
+    Enum.into map, %{}, fn {k, v} ->
+      if is_binary(k) && Enum.member?(params, k) do
```

Comment:
I don't think we need to check that they key is binary, we can just check with any key if it's a member of our forbidden/to be filtered parameter keys, worst case it returns false :)